Dangerfile

```diff
@@ -1,13 +1,40 @@
-# Sometimes it's a README fix, or something like that - which isn't relevant for
-# including in a project's CHANGELOG for example
-declared_trivial = github.pr_title.include? "#trivial"
+message "Welcome to danger"
 
-# Make it more obvious that a PR is a work in progress and shouldn't be merged yet
-warn("PR is classed as Work in Progress") if github.pr_title.include? "[WIP]"
+# # Sometimes it's a README fix, or something like that - which isn't relevant for
+# # including in a project's CHANGELOG for example
+# declared_trivial = github.pr_title.include? "#trivial"
 
-# Warn when there is a big PR
-warn("Big PR") if git.lines_of_code > 500
+# # Make it more obvious that a PR is a work in progress and shouldn't be merged yet
+# warn("PR is classed as Work in Progress") if github.pr_title.include? "[WIP]"
 
-# Don't let testing shortcuts get into master by accident
-fail("fdescribe left in tests") if `grep -r fdescribe specs/ `.length > 1
-fail("fit left in tests") if `grep -r fit specs/ `.length > 1
+# # Warn when there is a big PR
+# warn("Big PR") if git.lines_of_code > 500
+
+# # Don't let testing shortcuts get into master by accident
+# fail("fdescribe left in tests") if `grep -r fdescribe specs/ `.length > 1
+# fail("fit left in tests") if `grep -r fit specs/ `.length > 1
+
+newNoEncryptOptions = 0
+git.diff.each do |chunk|
+    if chunk.path.match(/src\/el/)
+      chunk.patch.lines.grep(/^+/).each do |added_line|
+          if added_line.match(/NoEncrypt:\s*true/)
+            newNoEncryptOptions = newNoEncryptOptions + 1
```

Comment:
the occurrences of newNoEncryptOptions should also reset itself at the end

Reply:
> the occurrences of newNoEncryptOptions should also reset itself at the end

This will be added